components/Album/Album.test.js

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from "@testing-library/react";
+import Album from "./Album";
+import userEvent from "@testing-library/user-event";
+
+const testPhotos = [
+  "/images/Carolaschlösschen1.jpg",
+  "/images/Carolaschlösschen2.jpg",
+  "/images/Carolaschlösschen3.jpg",
+];
+
+test("Image has alt text", () => {
+  render(<Album photos={testPhotos} />);
+  const photo = screen.getAllByRole("img");
```

Comment:
There is only one image on this page, isn't it? I think it should work with `getByRole` as well.